Umbraco-Cloud/Set-Up/Manage-Subscriptions/index.md

```diff
@@ -47,6 +47,14 @@ Changing subscription ownerships is not something you can do by yourself. You wi
 
 Whomever is taking over a subscription will need to have an existing account on umbraco.com. If they do not have an account they can [register here](https://shop.umbraco.com/profile/register).
 
+When you have registed your user. You will need login to your [profile] (https://shop.umbraco.com/profile/edit-your-details/) and fiil out the following info under the invocing information section.
```

Comment:
```suggestion
When you have registered your user. You will need to login to your [profile] (https://shop.umbraco.com/profile/edit-your-details/) and fill out the following info under the invoicing information section. Please make sure you fill out the following:
```

Reply:
@sofietoft This is super cool, thanks for the collaboration on this